### PR TITLE
screen-inhibit-control: add methods for delaying screensavers

### PR DESCRIPTION
**Pull request by 3v1n0**
Allow to call SimulateUserActivity on various supported screensavers.

### CODE REVIEW

**Comment by 3v1n0** [comment]
Probabyl this could be moved to `unity7`, but I though this place was more appropriate.

It might be the case to allow also `GetActive`, `GetActiveTime` and `GetSessionIdleTime`, while things such as `Lock` and `SetActive` would probably need a different interface.

@jdstrand what you think?

File: interfaces/builtin/screen_inhibit_control.go [hunk 1]
```diff
@@ -55,10 +55,19 @@ dbus (send)
...
     path=/Screensaver
     interface=org.freedesktop.ScreenSaver
-    member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit}
+    member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit,SimulateUserActivity}
+    peer=(label=unconfined),
+
+# gnome, kde and cinnamon screensaver
+dbus (send)
+    bus=session
+    path=/{ScreenSaver,}
```
Comment (jdstrand) [inline interfaces/builtin/screen_inhibit_control.go]: Can yoy change this to be: `path=/{,ScreenSaver}`. The rule is equivalent but follows the rules style we've established.

File: interfaces/builtin/screen_inhibit_control.go [hunk 2]
```diff
@@ -55,10 +55,19 @@ dbus (send)
...
+    interface=org.{gnome.ScreenSaver,kde.screensaver,cinnamon.ScreenSaver}
+    member=SimulateUserActivity
     peer=(label=unconfined),
 `
 
+
```
Comment (jdstrand) [inline interfaces/builtin/screen_inhibit_control.go]: Can you remove this extra newline?

**Comment by jdstrand** [comment]
@3v1n0 - SimulateUserActivity I think should be here, because that is a way of inhibiting the screensaver.

As for "GetActive, GetActiveTime and GetSessionIdleTime" would have to look at them more (a link to the dbus api stating what they do in a followup PR that adds them would be handy), but otoh, seems reasonable. I agree Lock and SetActive would probably need a different interface.

**Comment by 3v1n0** [comment]
Fine @jdstrand, updated. Thanks for your checks.

**Pull request merged**